main.c

```diff
@@ -508,13 +509,18 @@ main(int argc, char **argv)
 		"  --border-color <#argb>     Define border color\n"
 		"  --background-color <#argb> Define background color\n"
 		"  --bar-color <#argb>        Define bar color\n"
+		"  -f, --allow-overflow       Allow values over maximum\n"
```

Comment:
I'm thinking about enabling overflow by default and having only one long opt `--overflow-style=<nowrap|wrap>`. Most users expect wob not to exit on greater that maximum value anyways.

My reasoning behind `--overflow-style` over `--no-wrap` is that I would like to add overflow mode combining both `wrap` and `nowrap` - the "white" bar kept at maximum with the other "overflow" bar changing.

Reply:
Will do. A friend of mine who saw this suggested that same idea (of having it draw over itself) but I figured if you wanted that you could add it yourself.

Reply:
Do you want it to default to `wrap` or `nowrap`?

I was thinking to have it default to `wrap`, and then can be set to either `nowrap` or `none` with `--overflow-style`.

This way people can still turn it off if they really hate the overflow for some reason.

Reply:
Yes. Make it default to `wrap`.